spec/features/comments_feature_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+feature 'adding comments' do
+
+  let!(:sea) {Photo.create(title:'Awesome sea',
+                             image: File.new("spec/features/sea.jpg") )}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 6, 72 should be on its own line.<br>Space inside parentheses detected.